### DISCUSSION TITLE
posts/Go/ent/

### DISCUSSION THREAD

**Discussion by giscus** [Q1mi/BlogComments#270] [Announcements]
# posts/Go/ent/

李文周的 Blog 提供 Go 语言中文教程,李文周博客,Go 语言学习之路是我整理的一个免费教程，包含部分视频教程，也是非常适合新手学习Go语言的入门教程。本文介绍了 facebook 开源的 Go 语言 ORM框架 ent，包含schema定义及代码生成，并通过实例演示了如何进行数据增删改查及数据库迁移 

https://www.liwenzhou.com/posts/Go/ent/

**Comment by FlameCCG** [comment]
test
